src/AzureAuth.Test/CommandAadTest.cs

```diff
@@ -444,40 +444,6 @@ public void TestEvaluateOptionsWithoutAliasValidCommandLineOptions()
             subject.TokenFetcherOptions.Should().BeEquivalentTo(expected);
         }
 
-        [Ignore(reason: "No way to inject authflow yet")]
-        public void TestSendEvent_From_AuthFlowResult_With_Errors_And_Null_TokenResult()

```

Comment:
We want to get rid of this test all together?

Reply:
Yes, so the way this test was written is no longer relevant, and we've increased the test coverage of all the components that CommandAad is using. We are setup to write integration tests for this now if we want to, and we can come back and add more of this as needed.